test/system/090-events.bats

```diff
@@ -362,33 +381,38 @@ EOF
     run_podman rm $cname
 }
 
+# bats test_tags=ci:parallel
 @test "events - container inspect data - journald" {
     skip_if_remote "remote does not support --events-backend"
     skip_if_journald_unavailable
 
     _events_container_create_inspect_data journald
 }
 
+# CANNOT BE PARALLELIZED - #23750, events-backend=file cannot coexist with journal
 @test "events - container inspect data - file" {
     skip_if_remote "remote does not support --events-backend"
 
     _events_container_create_inspect_data file
 }
 
+# bats test_tags=ci:parallel
 @test "events - docker compat" {
-    local cname=c$(random_string 15)
-    t0=$(date --iso-8601=seconds)
+    local cname=c-$(safename)
+    t0=$(date --iso-8601=ns)
     run_podman run --name=$cname --rm $IMAGE true
+    # FIXME FIXME FIXME, please confirm that my 'container=' change is correct
```

Comment:
Eyeballs requested on this change, please

Reply:
I don't know what the indentation was but yes this seems correct status=$cname clearly makes no sense